packages/shopify-codemod/test/fixtures/iife-to-ternary-expression/basic.output.js

```diff
@@ -0,0 +1,30 @@
+(pass ? bar : baz);
```

Comment:
For sanity tests, can you add tests that this transform still works when the IIFE is part of a return statement/ part of an assignment expression?